src/gradletestrunnersmoketest.sh

```diff
@@ -18,14 +18,25 @@ gradletestrunnersmoketest () {
   ./gradlew spotlessCheck
   ./gradlew spotbugsMain
 
-  echo "Running test suite"
-  ./gradlew runTest --args="suites/PRSmokeTests.json tmp/TestRunnerResults"
+  local outputDir="/tmp/TestRunnerResults"
```

Comment:
```suggestion
  outputDir="/tmp/TestRunnerResults"
```

Is this variable used in other places?

Reply:
I'm not sure. But it seems safer to scope it locally rather than modify the global context.